src/code/code_800EC960.c

```diff
@@ -4278,19 +4276,17 @@ void Audio_SetMainBgmVolume(u8 targetVol, u8 volFadeTimer) {
  * Incrementally increase volume of NA_BGM_GANON_TOWER for each new room during the climb of Ganon's Tower
  */
 void Audio_SetGanonsTowerBgmVolumeLevel(u8 ganonsTowerLevel) {
-    u8 channelIdx;
-    s8 pan = 0;
+    u8 channelIndex;
+    s8 panChannelWeight = 0; // Pan comes entirely from the SequenceLayer
 
     // Ganondorf's Lair
     if (ganonsTowerLevel == 0) {
-        pan = 0x7F;
+        // Pan is split evenly between the SequenceChannel and SequenceLayer
+        panChannelWeight = 0x7F;
```

Comment:
are you sure?
I'm simply looking at `Audio_SequenceChannelProcessSound` but it looks like that would result in `( channel->newPan * 0x7F + layer->pan * (0x80 - 0x7F) ) >> 7 ~= channel->newPan`

Reply:
Yeah, you're right here